packages/cli/package.json

```diff
@@ -166,7 +167,7 @@
     "tar-fs": "1.16.3",
     "title": "3.4.1",
     "tldts": "6.1.47",
-    "tmp-promise": "1.0.3",
```

Comment:
**question (non-blocking):** was there no interface or behavior change between these versions?

I couldn't find a changelog for these releases. Are we sure this change is OK?‏

Reply:
Went back to `1.0.3` and found another solution to be safe.